contrib/docker/start.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/local/bin/python
+
+import jinja2
+import os
+import sys
+import subprocess
+import glob
+
+convert = lambda src, dst, environ: open(dst, "w").write(jinja2.Template(open(src).read()).render(**environ))
+mode = sys.argv[1] if len(sys.argv) > 1 else None
+environ = os.environ.copy()
+
+# Check mandatory parameters and build the base start arguments
+if "SYNAPSE_SERVER_NAME" not in environ:
```

Comment:
Doesn't this mean that one can never remove this env var? Even after the generation of the config?

Reply:
Yep, this got to be improved, will include the fix while making some other options mandatory. The expected behaviour would be: server name is mandatory for generating the config file, some variables are mandatory when using the dynamic config.